SB2018/2018-01-12-2.md

```diff
@@ -0,0 +1,34 @@
+---
+to: Southend
+subject: SourceBots Mentoring and Tech Days
+---
+
+Hi all,
+
+We hope you're progressing well on your robot!
+We have some important things to announce, so keep reading:
+
+## Mentoring
+
+We're not sure if we can provide mentoring for you, as you are quite far away from most of our possible mentors, however if you really need someone to mentor you, we'll see what we can do.
+
+If you're interested in having a mentor come and help you, **reply to this email with the times your teams meet**, and we'll see if we can assign someone to you!
+
+## Tech Days
+
+We're happy to announce some tech days!
+
+They'll be at the University of Southampton Electronics and Computer Science department in the state-of-the-art electronics labs, with a full mechanical workshop.
+
+Tech days start at 10:00am and finish at 5:00pm. They are a great opportunity for you to get robotics advice and help with technical issues from our volunteers. We typically give a tour of the university facilities as well.
+
+Here are the dates:
+
+    - 03 Feb 2018
+    - 17 Feb 2018
+    - 17 Mar 2018
+    - 14 Apr 2018
+
+
+Thanks,
```

Comment:
Can we remove the signature at the bottom. All other emails have it removed, and the chances of me forgetting to remove it are far too high!

Reply:
Alternatively, we could sign them off "The SourceBots Team" which wouldn't matter if it was accidentally left in place.

Reply:
I think that'd work nice if they were sent from `teams@`, however currently it's from our personal accounts (eg `jhoward@`). Moving to `teams@` would be a nice idea, but OOS for this discussion